homeassistant/components/adguard/services.yaml

```diff
@@ -28,6 +34,12 @@ remove_url:
       example: https://www.example.com/filter/1.txt
       selector:
         text:
+    list_type:
+      name: List type
+      description: The list type (blocklist/allowlist) of the filter you are removing.
+      default: blocklist
+      selector:
+        text:
```

Comment:
You don't need list_type for remove

Reply:
You are correct. I added this comment because it didn't match `SERVICE_REMOVE_URL_SCHEMA`
I don't think you need `SERVICE_REMOVE_URL_SCHEMA` and you can always use `SERVICE_ADD_URL_SCHEMA`

Reply:
@epenet 
I think we do need `SERVICE_REMOVE_URL_SCHEMA` because `vol.Required(CONF_NAME): cv.string,`
So `CONF_NAME` is required with `ADD_URL` and isn't needed with `REMOVE_URL`

Reply:
@gieljnssns I have reviewed this again, and I think the reason for the confusion is that previous implementation was wrong. CONF_NAME was required by the schema, but ignored by the service.
This is good now.